src/main/java/org/geomerty/procedural/shapes/Triangle.java

```diff
@@ -0,0 +1,7 @@
+package org.geomerty.procedural.shapes;
+
+public class Triangle {
+    public double a;
```

Comment:
@EhsanMaqs here you see have many ways there are to represent a triangle. Other students had `base` and `height` fields for triangle. I know another way to define a triangle: adjacent sides and angle between them. How great that objects encapsulate all those details and let you make changes without breaking the rest of the code.